### PR TITLE
Incorporate feedback from hands-on

### PR DESCRIPTION
**Pull request by titosanxez**

### CODE REVIEW

File: examples/routing_service_shapes_processor/cpp/README.rst [hunk 1]
```diff
@@ -21,6 +21,7 @@ From the directory containing the example sources:
     cd build
     cmake -DCONNEXTDDS_DIR=<Connext directory> \
             -DCONNEXTDDS_ARCH=<ARCH> \
+            -DCMAKE_RUNTIME_OUTPUT_DIRECTORY
```
Comment (iblancasa) [inline examples/routing_service_shapes_processor/cpp/README.rst]: This needs a value and is something you  can set as a target property

Comment (titosanxez) [reply examples/routing_service_shapes_processor/cpp/README.rst]: Good catch, I forgot to update this properly. Will this be enough for Visual Studio?

File: examples/routing_service_shapes_processor/cpp/CMakeLists.txt [hunk 2]
```diff
@@ -33,16 +44,27 @@ find_package(
 add_library(shapesprocessor
     "${CMAKE_CURRENT_SOURCE_DIR}/ShapesProcessor.cxx")
 
+# Set target properties for lang requirement output library name
+set_target_properties(shapesprocessor
+    PROPERTIES
+        CXX_STANDARD 11
+        OUTPUT_NAME_DEBUG shapesprocessord
+        LIBRARY_OUTPUT_DIRECTORY "${output_dir}"
```
Comment (iblancasa) [inline examples/routing_service_shapes_processor/cpp/CMakeLists.txt]: Do you need to specify all the configurations? I mean, if you set ``LIBRARY_OUTPUT_DIRECTORY`` you shouldn't need to set ``LIBRARY_OUTPUT_DIRECTORY_RELEASE`` . If ``LIBRARY_OUTPUT_DIRECTORY_RELEASE`` is not set, CMake will try to use ``LIBRARY_OUTPUT_DIRECTORY``.

Comment (titosanxez) [reply examples/routing_service_shapes_processor/cpp/CMakeLists.txt]: We discussed this online and we agreed we need them

**Pull request merged**